### DISCUSSION TITLE
How to launch or find some of the titles listed in the mame whatsnew.txt

### DISCUSSION THREAD

**Discussion by paulmortlock** [sam-ludlow/mame-ao#23] [Q&A]
So in the mame whatsnew ...

spectrum_cass:
  Anti-Lune [Michael Bruhn, Planeta Sinclair]
  Rodmän [Mika Keranen]
  Tokimal (demo) [Pat Morita Team]
  Cálculo de Areas e Volumes, Cálculo de Estrutura Tronco-Cónicas, Foguetão, Geografia de Portugal (Astor),
  Mad Mix Game & The Pepsi Challenge (Portuguese), Pro Golf II (System 4, alt), Pssst (Nuova Newel Software),
  Pssst + Jetpac + Kucharik, Sachy Psion + Reversi + Backgammon, O Segredo dos Templários (TAP tape image), Starblazer,
  Tabela Periódica, Tabela Periódica (fast version hack) [Planeta Sinclair]
  Black & White, Box Reloaded, Flunky (Zafi Chip), GP Motocross, Left Behind: Escape From Mars, Mad Cars, Pot-Pourri,
  Quest for the Sacred Flame of Hestia, Space Racer (Proein), Stop Virus (demo) [Spectrum Computing]
  La Balada del Duende, Editor Musical, La Isla Maldita Juego de Damas [El Trastero del Spectrum]

spectrum_cass:
  Anti-Lune [Michael Bruhn, Planeta Sinclair]
  Rodmän [Mika Keranen]
  Tokimal (demo) [Pat Morita Team]
  Cálculo de Areas e Volumes, Cálculo de Estrutura Tronco-Cónicas, Foguetão, Geografia de Portugal (Astor),
  Mad Mix Game & The Pepsi Challenge (Portuguese), Pro Golf II (System 4, alt), Pssst (Nuova Newel Software),
  Pssst + Jetpac + Kucharik, Sachy Psion + Reversi + Backgammon, O Segredo dos Templários (TAP tape image), Starblazer,
  Tabela Periódica, Tabela Periódica (fast version hack) [Planeta Sinclair]
  Black & White, Box Reloaded, Flunky (Zafi Chip), GP Motocross, Left Behind: Escape From Mars, Mad Cars, Pot-Pourri,
  Quest for the Sacred Flame of Hestia, Space Racer (Proein), Stop Virus (demo) [Spectrum Computing]
  La Balada del Duende, Editor Musical, La Isla Maldita Juego de Damas [El Trastero del Spectrum]

How do you find how to launch via mame-ao any of these. I can see the XML file  in the Hash folder but am stuck. I would like to be able to find all the titles and launch somehow for a particular system.

**Comment by sam-ludlow** [comment]
Hello @paulmortlock 

Ensure you are searching everything.

Then you need to get a bit creative with what exactly you search for from the whatsnew.txt

MAME-AO search isn't that smart it just looks if the name or description contains the search text, wrong way round or include a word not in the name and description you will not find.

For example here

- "Excalibur Electronics Igor" - won't find Jack
- "Igor Excalibur" - works (or just ("Igor")

![image](https://github.com/sam-ludlow/mame-ao/assets/47115020/b38b5d74-a85d-4d45-a4f4-b11ad525b843)

Cheers
Sam

**Comment by sam-ludlow** [comment]
With software you need to find a machine first

for example "ZX Spectrum" there may be multiple software lists make sure you're on the right one.

![image](https://github.com/sam-ludlow/mame-ao/assets/47115020/97756a96-d39e-4eb2-bbed-2ebf1cba4206)

**Comment by paulmortlock** [comment]
Ah I see thanks. Is there room for improvement in this area.
